Flabbergasted-Flamingos/bangazon-commerce-app
# Jccreate product

# Description
When adding a product to sell you can now add the product type
When looking at your list of products you are afforded the option to delete them

## Related Ticket(s)
close #16 
close #11 
close #7 

## Steps to Test Solution

1. git fetch --all
2. git checkout JCcreateProduct
3. delete migrations and db
4. python manage.py makemigrations
5. python manage.py migrate
5b. go into the db and create a product type entity 
6. python manage.py runserver
7. go to localhost and create an account
8. got to localhost/sell
9. Create a product to sell and make sure the product type you create in 5b shows up in the product type drop down
10. After getting the successfully added page click on the product list
11. This view should show the product you just create and give the option to delete it
12. Click delete
13. After delete it should take you back to the products page
